packages/reporter-vue/src/commands/Command.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <CommandRow :size="size" class="command-row" :class="{ [command.state]: true, selected, 'event': command.event, scaled }" @select="selected = $event">
+    <template #position="{ hovering }">
+      <div class="command-position">
+        <i-mdi-pin v-if="hovering || selected"></i-mdi-pin>
+        <i-fa-spinner v-else-if="command.state === 'pending'" class="w-14px h-14px animate-spin"/>
+        <span v-else-if="!command.event === true">{{ idx + 1 }}</span>
+      </div>
+    </template>
+    <template #name="{}">
+      <span class="command-name font-semibold">{{ commandName }}</span>
+    </template>
+    <template #message="{}">
+      <span class="command-message">
+        <i-fa-circle v-if="command.renderProps && command.renderProps.indicator" class="indicator" :class="command.renderProps && command.renderProps.indicator"/>
+        <span class="message">
+          <Markdown v-if="showMarkdown" :source="commandMessage"></Markdown>
+          <span v-if="!showMarkdown">{{ commandMessage }}</span>
+        </span>
+      </span>
+    </template>
+    <template #meta="{}">
+      <span class="command-meta">
+        <i-fa-eye-slash v-if="command.visible === false"/>
+        <span class="bg-warm-gray-400 px-4px py-4px rounded-md  text-size-10.2px text-white" v-if="command.numElements >= 0">
+          {{ command.numElements }}
+        </span>
+      </span>
+    </template>
+  </CommandRow>
+</template>
+
+<script lang="ts">
+import { defineComponent, computed, PropType, ref } from 'vue'
+import Markdown from 'vue3-markdown-it'
+import CommandRow from './CommandRow.vue' 
+import CommandMessage from './CommandMessage.vue'
+export default defineComponent({
+  props: {
+    command: {},
+    idx: {},
+    size: {
+      values: ['sm', 'md', 'lg'],
```

Comment:
What is `values`? I have never seen this syntax, shouldn't this be something like

```ts
size: {
  type: String as () => PropType<'sm' | 'md' | 'lg'>
}
```

Reply:
Yes, it should.